src/movies.js

```diff
@@ -1,15 +1,95 @@
 /* eslint no-restricted-globals: 'off' */
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
+const orderByYear = movies =>
+  [...movies].sort((firstEl, secondEl) => {
+    if (firstEl.year > secondEl.year) return 1;
```

Comment:
You actually don't have to return 1 or -1, etc. what you instead can write is a number that is either greater or lower than 0. With this is mind what you can do is just return this:

.sort(
 if (firstEl.year === secondEl.year) return firstEl.title.localeCompare(secondEl.title);
 return firstEl.year - secondEl.year
)

This is because firstEl.year - secondEl.year will return a negative number, a positive number, or 0